src/VisualStudio/Setup.Next/source.extension.vsixmanifest

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We'll need to figure out how to appropriately insert this into main VS builds for VS "15", but not for VS 2015.  Maybe @jmarolf can help me with that part?